src/pybind/CMakeLists.txt

```diff
@@ -67,6 +67,23 @@ if(WITH_MGR)
   elseif(NOT WITH_PYTHON3 AND MGR_PYTHON_VERSION_MAJOR EQUAL 3)
     message(FATAL_ERROR "mgr plugins require python3 binding")
   endif()
+
+  if (USE_OPENSSL)
+    execute_process(
+      COMMAND ${PYTHON${PYTHON_VERSION}_EXECUTABLE} -c "import ssl; print('.'.join(map(str,ssl.OPENSSL_VERSION_INFO[0:3])))"
+      RESULT_VARIABLE PYSSL_RESULT
+      OUTPUT_VARIABLE PYSSL_VER
+      ERROR_QUIET
```

Comment:
nit, move `)` to the previous line. the reason we have `)` in its own line is to avoid extra code churn when adding new element(s) in a cmake command.

Reply:
@theanalyst could you re-consider this comment?